eng/test-determinism.ps1

```diff
@@ -32,7 +32,11 @@ $script:skipList = @(
   "Roslyn.VisualStudio.DiagnosticsWindow.dll.key",
 
   # Work around resx issues https://github.com/dotnet/roslyn/issues/77544
-  "Text.Analyzers.dll.key"
+  "Text.Analyzers.dll.key",
+
+  # The integration test framework uses a VSIX as an embedded resource in this dll.
+  # The VSIX build is not deterministic (for example the catalog.json contains a randomly generated identifier for the install location).
+  "Microsoft.VisualStudio.Extensibility.Testing.Xunit.dll"
```

Comment:
Is this worth having a tracking bug for this? Or is this mechanism something we'd eventually get rid of in some other way? I'm torn between "we could eventually make this work" and "this will never work, this is totally fine to have the exception".